formidable/forms/conditions.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import six
+
+
+class ConditionsRegister(dict):
+
```

Comment:
Broken windows theory.
I started this using Presets as an example, but there's not a lot of docstring there...